cli-impl/src/test/resources/negative/GoalAndMeta.txt

```diff
@@ -190,6 +190,15 @@ Error: Unable to solve the type of this literal:
 Let's learn from that.
 
 NonPattern:
+In file C:\Users\hoshino\Documents\Projects\project-aya\aya-dev\cli-impl\src\test\resources\shared\src\data\vec\base.aya:5:6 ->
```

Comment:
Why is there absolute path?

Reply:
Ah, I think it's related to shadowing warnings.

Reply:
omg